tests/core/test_wprintf.out

```diff
@@ -27,6 +27,9 @@ vswoutput st-rts with 0x74
 vsw    continues with 0x65
 vsw    continues with 0x73
 test string has 36 wide characters.
+PrintBigWide wrote 426 wchars:
+test string has 425 wide characters.
```

Comment:
Note: difference in the above is expected, as "%d" is 2 chars, get gets printed as "425", which is 3 chars.